enhancements/two-nodes-openshift/2no.md

```diff
@@ -216,7 +239,12 @@ This will allow the use of a credential scoped to `ConfigMap`s in the `openshift
 
 The plan is for this to be changed by one of the nodes during pacemaker initialization. Pacemaker initialization should be initiated by CEO when it detects that the cluster controlPlane topology is set to `DualReplica`.
 
-#### Install Config with Fencing Credentials
+While set to `External`, CEO will still need to render the configuration for the etcd pod in a place where it can be consumed by pacemaker. This ensures that the pod
+can be updated with a new container reference in the case of a upgrade event or whenever certificates are rotated.
```

Comment:
nit: pacemaker is running a container only, not a pod (currently at least, @clobrano please keep me honest here). What about "This ensures that the etcd instance managed by pacemaker can be updated accordingly in the case of a upgrade event or whenever certificates are rotated."

Reply:
> nit: pacemaker is running a container only, not a pod (currently at least, @clobrano please keep me honest here)

Correct, and also I like very much Marc's suggested version, it is clear :+1:

Reply:
Yes - this is a good clarification. The point is that is it's not running in cluster, so calling it a pod is inaccurate and confusing

Reply:
podman can run pods as well, without any cluster 😉

Reply:
This is news to me! Very cool!